inc/storefront-template-functions.php

```diff
@@ -165,7 +165,7 @@ function storefront_site_branding() {
 			jetpack_the_site_logo();
 		} else { ?>
 			<div class="site-branding">
-				<h1 class="site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></h1>
+				<div class="beta site-title"><a href="<?php echo esc_url( home_url( '/' ) ); ?>" rel="home"><?php bloginfo( 'name' ); ?></a></div>
```

Comment:
And i just notice that Yoast plugin also generate website structured data on front page. So not sure we need it ?